linera-chain/src/chain.rs

```diff
@@ -490,8 +490,11 @@ where
         self.execution_state.system.is_active()
     }
 
-    /// Invariant for the states of active chains.
-    pub async fn ensure_is_active(&mut self, local_time: Timestamp) -> Result<(), ChainError> {
+    /// Initializes the chain if it is not active yet.
+    pub async fn initialize_if_inactive(
```

Comment:
Maybe just _initialize_ ? and a comment that this is idempotent and if it's already initialized it returns early.

Reply:
`maybe_initialize` ?

Reply:
Done in https://github.com/linera-io/linera-protocol/pull/4646/commits/b678c1e3f062387a9636beeb7f6ab7decc42cd99.

Reply:
I don't like `maybe_` in method names. `initialize` is idempotent function anyway and the caller doesn't care if it's the first or second (or any other) call to it. It just cares it's initialized.

Anyway, not a blocker.